mta/mta_services_test.go

```diff
@@ -321,9 +293,52 @@ var _ = Describe("MtaServices", func() {
 			Ω(resources).Should(BeNil())
 		})
 	})
+
+	var _ = Describe("isNameUnique", func() {
+		It("Check if name exists in mta.yaml", func() {
+			mtaPath := getTestPath("result", "mta.yaml")
+
+			jsonRootData, err := json.Marshal(oMtaInput)
+			Ω(err).Should(Succeed())
+			Ω(CreateMta(mtaPath, string(jsonRootData), os.MkdirAll)).Should(Succeed())
+
+			oModule.Provides = append(oModule.Provides, oProvides)
+			jsonModuleData, err := json.Marshal(oModule)
+			Ω(err).Should(Succeed())
+			Ω(AddModule(mtaPath, string(jsonModuleData), Marshal)).Should(Succeed())
+			oMtaInput.Modules = append(oMtaInput.Modules, &oModule)
+
+			jsonResourceData, err := json.Marshal(oResource)
+			Ω(err).Should(Succeed())
+			Ω(AddResource(mtaPath, string(jsonResourceData), Marshal)).Should(Succeed())
+			oMtaInput.Resources = append(oMtaInput.Resources, &oResource)
+
+			Ω(mtaPath).Should(BeAnExistingFile())
+
```

Comment:
why do we need all this long story? We can use ready mta.yaml and ask for unique name from it